testing/test_sample.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from conceptnet_lite import Label, Language, edges_for, edges_between, connect, Concept, Relation
+
+
+@pytest.fixture(scope="session")
+def _connect_to_conceptnet():
```

Comment:
how about connecting to the database once and then running all the tests from there, rather than reconnecting for each test?

Reply:
@annargrs `@pytest.fixture(scope="session")` does exactly this.